testing/src/main/java/org/jdbi/v3/testing/junit5/JdbiPostgresExtension.java

```diff
@@ -63,8 +63,7 @@ public class JdbiPostgresExtension extends JdbiExtension {
 
     private volatile DatabaseInfo info;
 
-    static final JdbiExtension instance(EmbeddedPgExtension pg) {
```

Comment:
this seems unrelated?

Reply:
or maybe needs a rebase?

Reply:
Not part my PR. Not sure how Github crunches the data to produce this?!
I will rebase the pull request.